tests/external/test_report_downloader.py

```diff
@@ -63,3 +63,27 @@ def test_invalid_provider_type(self):
                                  report_source='hereiam',
                                  report_name='bestreport',
                                  provider_type='unknown')
+
+    @patch('masu.external.downloader.aws.aws_report_downloader.AWSReportDownloader.__init__', return_value=None)
+    def test_get_current_report(self, fake_downloader):
+        """Test get_current_report function."""
+        downloader = ReportDownloader(customer_name='customer name',
+                                      access_credential='mycred',
```

Comment:
I've added a new facility to generate a fake ARN. Please use that here.

I had to squash a number of bugs in the tests because we weren't passing ARNs into the credentials.